<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QMainWindow" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Segoe Script</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Inicio</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Forte</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Configuraciones:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <family>Noto Sans Lao</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Temporizador</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="temporCombo">
                  <item>
                   <property name="text">
                    <string>Temporizador por juego</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Temporizador por jugada</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Sin temporizador</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <family>Noto Sans Lao</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Nivel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="nivelComobo">
                  <item>
                   <property name="text">
                    <string>Facil</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Medio</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dificil</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <family>Noto Sans Lao</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Repetible</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="repetibleCheckBox">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="salir">
            <property name="text">
             <string>Cerrar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="iniciarJuego">
            <property name="text">
             <string>Iniciar Juego</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
